sources/packages/workflow/BPMN/camunda-8/parttime-configure-disbursement.bpmn

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0uxv5b2" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.4.2" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.1.0">
+  <bpmn:process id="parttime-configure-disbursement" name="parttime-configure-disbursement" isExecutable="true">
+    <bpmn:exclusiveGateway id="Gateway_0hiv6nu">
+      <bpmn:incoming>Flow_16jouca</bpmn:incoming>
+      <bpmn:outgoing>Flow_0pl9xpo</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0mhntn4</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_012lvqx" name="disbursmenentScheduleDate1">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=today()" target="disbursmenentScheduleDate1" />
```

Comment:
is this a inbuild date library

Reply:
https://docs.camunda.io/docs/components/modeler/feel/builtin-functions/feel-built-in-functions-temporal/